src/utils/filter.tsx

```diff
@@ -0,0 +1,70 @@
+import { Person } from '../types';
+
+export enum FilterTypes {
+  All = '',
+  Male = 'm',
+  Female = 'f',
+}
+
+export const filter = (
+  selectedFilter: URLSearchParams,
+  peopleList: Person[],
+): Person[] => {
+  let preparedPeople = peopleList;
+  const sexTypes = selectedFilter.get('sex');
+  const queryFilter = selectedFilter.get('query');
+  const centuryFilter = selectedFilter.getAll('centuries');
+  const sort = selectedFilter.get('sort');
+  const order = selectedFilter.get('order');
```

Comment:
can we use enum instead of magic string?

Reply:
Done